articles/encrypting-gdrive-using-rclone/index.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/encrypting-gdrive-using-rclone/
+title: How to Encrypt Google Drive using Rclone
+description: A how-to guide on setting up Rclone to create an encrypted folder where users can store data on Google Drive which will be encrypted before upload.
+author: louise-findlay
+date: 2020-08-25T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/encrypting-gdrive-using-rclone/hero.png
+    alt: rclone gdrive header image
+---
+This is a how-to guide on setting up Rclone to create an encrypted folder where users can store data on Google Drive which will be encrypted before upload.
```

Comment:
It seems this article is specifically about Linux and won't be directly applicable for Mac or Windows users, perhaps make this clear early.

Reply:
I've added a note in the intro mentioning it's Linux/Mac only.